README.md

```diff
@@ -139,3 +139,11 @@ The route optimization client is in the maven central repository, thus you can j
 ## Customization
 
 If you want to build it from source please follow the guide for java [here](https://github.com/graphhopper/directions-api-clients-route-optimization).
+
+# Geocoding
+
+## Generate from Swagger
+
+```
+java -jar ../swagger-codegen-cli.jar generate -i geocoding_swagger.yaml -l java -o geocoding -c geocoding-config.json
```

Comment:
We have to streamline this somehow with the route opt client (but can do this later on). E.g. using the same [config stuff](https://github.com/graphhopper/directions-api-clients-route-optimization/blob/master/create.sh#L24-L25) e.g. it overwrite model and package names.